_dashboards/install/index.md

```diff
@@ -10,3 +10,20 @@ redirect_from:
 # Install and configure OpenSearch Dashboards
 
 OpenSearch Dashboards has three installation options at this time: Docker images, tarballs, and Helm charts.
+## Using sample data
+
+To get a first look at how OpenSearch Dashboards works, you can explore using the following sample data:
+
+* Global flight data
+* Revenue data
+* Web traffic data
+
+In this tutorial you will create a simple dashboard using the sample flight data.
+
+1. Start OpenSearch Dashboards. You can access it at port 5601. For example, http://localhost:5601.
+1. Log in with the default username `admin` and password `admin`.
```

Comment:
This assumes the user installed the security plugin. In the minimal distribution, there's no login process.